### PR TITLE
Add documentation for lidspace compaction tuning

### PR DESCRIPTION
**Pull request by baldersheim**
@kkraune or @geirst PR

### CODE REVIEW

File: en/reference/services-content.html [hunk 1]
```diff
@@ -910,6 +918,24 @@ <h2 id="initialize">initialize</h2>
...
+</p>
+<ul>
+  <li id="lidspace-max-bloat-factor"><code>max-bloat-factor</code>:
+    Maximum bloat allowed before lidspace compaction is started. Compaction is moving a document
+    from a high lid to a lower lid. Cost is similar to feeding a document and removing it.
+    Default value is 0.01 or 1% of total lidspace. Will be increased to target of 0.50 or 50%.
+  </li>
+</ul>
+<pre>
+&lt;initialize&gt;
```
Comment (geirst) [inline en/reference/services-content.html:934]: Please update example with correct syntax.

**Review by geirst — COMMENTED** [review]
👍

File: en/reference/services-content.html [hunk 2]
```diff
@@ -910,6 +918,24 @@ <h2 id="initialize">initialize</h2>
...
+<h2 id="lidspace">lidspace</h2>
+<p>
+Contained in <a href="#searchnode">searchnode</a>, optional.
+Tune settings related to how lidspace is managed. Optional sub-elements:
+</p>
+<ul>
+  <li id="lidspace-max-bloat-factor"><code>max-bloat-factor</code>:
+    Maximum bloat allowed before lidspace compaction is started. Compaction is moving a document
+    from a high lid to a lower lid. Cost is similar to feeding a document and removing it.
+    Default value is 0.01 or 1% of total lidspace. Will be increased to target of 0.50 or 50%.
```
Comment (kkraune) [inline en/reference/services-content.html:930]: I did not understand "Will be increased to target of 0.50 or 50%."

**Review by kkraune — APPROVED** [review]

File: en/reference/services-content.html (hunk 2 above)
Comment (baldersheim) [reply en/reference/services-content.html:930]: Default is today 0.01 or 1%. This is a very bad default, and will soon, at some point in time be adjusted to 0.50, or 50%.

File: en/reference/services-content.html (hunk 1 above)
Comment (baldersheim) [reply en/reference/services-content.html]: Fixed

File: en/reference/services-content.html [hunk 3]
```diff
@@ -910,6 +918,24 @@ <h2 id="initialize">initialize</h2>
 &lt;/initialize&gt;
 </pre>
 
+<h2 id="lidspace">lidspace</h2>
+<p>
+Contained in <a href="#searchnode">searchnode</a>, optional.
+Tune settings related to how lidspace is managed. Optional sub-elements:
```
[the remark this answers is not in the document]
Comment (baldersheim) [reply en/reference/services-content.html:924]: Fixed

File: en/reference/services-content.html (hunk 2 above)
Comment (baldersheim) [reply en/reference/services-content.html:931]: Forgot to reload ?
I see it.

**Pull request merged**